code/modules/projectiles/guns/projectile/siren.dm

```diff
@@ -8,12 +8,13 @@
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guninhands_left.dmi', "right_hand" = 'icons/mob/in-hand/right/guninhands_right.dmi')
 	recoil = 1
 	slot_flags = SLOT_BELT
-	flags = FPRINT | NOREACT
+	flags = FPRINT | NOREACT | OPENCONTAINER
```

Comment:
The *siren* has `NOREACT`.
I explicitly removed that flag from its children, the super soaker and squirt gun.